packages/client-ui/src/components/common/WorkflowStep.tsx

```diff
@@ -25,24 +26,15 @@ const WorkflowStep = ({ projectSteps, activeStep }: WorkflowStepProps): JSX.Elem
         </Typography>
 
         <Breadcrumbs aria-label="breadcrumb" className={classes.breadcrumbs}>
-          {[projectSteps[activeStep]?.step?.parentId, projectSteps[activeStep]?.step?.moduleType].map(
-            (path, index) => (
-              <Typography color="textPrimary" key={index}>
-                {path}
-              </Typography>
-            )
-          )}
+          <Typography color="textPrimary">
+            {currentPageDetails?.meta?.label ?? `Page ${currentPageDetails?.id}`}
```

Comment:
The removed code included the name of the workflow followed by the name of the step. The new code contains only the name of the step, but misses the name of the workflow.

Reply:
So `parentId` was used as the name of the workflow? That's really unclear and was probably incorrect in case of a graph with subbranches! I can add workflow label here.